LadderBots.json

```diff
@@ -2,14 +2,25 @@
 
 	"Bots": {
 		"CryptBot": {
-			"Race": "Terran",
+			"Race": "Protoss",
 			"Type": "BinaryCpp",
 			"Path": "bot_simple.exe"
 		},
-		"ProtossBot": {
+		"TerranBot": {
 			"Race": "Terran",
 			"Type": "BinaryCpp",
-			"Path": "bot_simple.exe"
+			"Path": "bot_simple.exe",
+			"Args": "-something"
```

Comment:
I suggest to add one more example arg in order to illustrate how a sequence of args could be passed.

Reply:
Yeah I was a bit tired at that point, thus the very "creative" -something.

Reply:
Your approach looks good to me too. Probably you should preserve it.
@Cryptyc ?